.baseline/findbugs/excludeFilter.xml

```diff
@@ -1,5 +1,9 @@
 <!-- See http://findbugs.sourceforge.net/manual/filter.html for syntax -->
 <FindBugsFilter>
+    <!-- ignore generated code -->
+    <Match>
+        <Class name="~.*\.*Immutable.*" />
```

Comment:
I'll try that locally and see if that resolves the issue. Issue I'm seeing right now is that default Eclipse projects generated currently report FindBugs warnings and Checkstyle errors.